Gravity/Gravity.Test.Integration/Gravity.Test.Integration.csproj

```diff
@@ -82,12 +82,15 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\..\Gravity\Gravity\Gravity.Test\Gravity.Test.csproj">
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
What caused the Service Include?

Reply:
https://stackoverflow.com/a/22917902/1180926